utils/server.py

```diff
@@ -0,0 +1,81 @@
+from aiohttp import web
+import socket
+from json import JSONDecodeError
+
+
+def _http_error_handler(error=False) -> web.Response:
+    """
+    Used to handle HTTP error response.
+    Parameters
+    ----------
+    error : bool, optional
+        Bool or string to be used, by default False
+    Returns
+    -------
+    web.Response
+        AIOHTTP web server response.
+    """
+
+    return web.json_response(
+        {"error": error},
+        status=400 if error else 200
+    )
+
+
+class WebServer:
+    def __init__(self, bot):
+        self.bot = bot
+        self.ctx = None
+        self.channels = None
+        self.players = None
+        self.IP = socket.gethostbyname(socket.gethostname())
+        self.port = 3000
+
+    async def _handler(self, request: web.Request) -> web.Response:
+        """
+        Super simple HTTP handler.
+        Parameters
+        ----------
+        request : web.Request
+            AIOHTTP request object.
+        """
+
+        # or "Authorization"
+        if request.method != "POST" not in request.headers:
```

Comment:
Not sure if this is what you meant as I believe it would check if True or False is in request.headers
```python3
if (True / False) not in request.headers:
    ....
```

Reply:
Fixed